src/main/java/com/univ/lille/copainsderoute/platine/controller/EventController.java

```diff
@@ -61,4 +62,24 @@ public ResponseEntity<String> deleteEvent(@PathVariable("id") int id) throws Run
         return ResponseEntity.ok("Event deleted");
     }
     
-}
+   @GetMapping("{northeastLatitude}/{northeastLongitude}/{southwestLatitude}/{southwestLongitude}/{northwestLatitude}/{northwestLongitude}/{southeastLatitude}/{southeastLongitude}")
+    public ResponseEntity<List<Event>> getEventsByLocation(
+            @PathVariable("northeastLatitude") Double northeastLatitude,
+            @PathVariable("northeastLongitude") Double northeastLongitude,
+            @PathVariable("southwestLatitude") Double southwestLatitude,
+            @PathVariable("southwestLongitude") Double southwestLongitude,
+            @PathVariable("northwestLatitude") Double northwestLatitude,
+            @PathVariable("northwestLongitude") Double northwestLongitude,
+            @PathVariable("southeastLatitude") Double southeastLatitude,
+            @PathVariable("southeastLongitude") Double southeastLongitude
+    ) throws RuntimeException {
+
+        GpsCoordinatesDTOs gpsCoordinatesDTOs = new GpsCoordinatesDTOs(northeastLatitude, northeastLongitude, southwestLatitude, southwestLongitude, northwestLatitude, northwestLongitude, southeastLatitude, southeastLongitude);
```

Comment:
Pourquoi ne pas mettre le DTO en paramètres de la méthode ? comme ça il est mappé direct et il faut juste mettre les données dans le body a la place

```suggestion
    public ResponseEntity<List<Event>> getEventsByLocation(@RequestBody GpsCoordinateDTO gpsCoordinates)
```